packages/apollo-voyager-tools/src/wrapResolvers.test.ts

```diff
@@ -0,0 +1,59 @@
+import auditLogger from '@aerogear/apollo-voyager-audit'
+import metrics from '@aerogear/apollo-voyager-metrics'
+import { voyagerResolvers } from '@aerogear/apollo-voyager-server'
+import test from 'ava'
+import { wrapResolvers } from './wrapResolvers'
+import { FieldResolver } from './wrapResolvers'
+
+function resolverFunction() {
+  return (): FieldResolver => {
+    return (obj, args, context, info) => {
+      return context
+    }
+  }
+}
+
+const resolversWithOutSubscription = {
+  Query: {
+    allTasks: async (context: any) => {
+      return context.db.select().from('tasks')
```

Comment:
It's kinda weird to see this code in our unit tests. Let's have a chat about how to test this stuff properly